modules/ROOT/pages/apikit-4-soap-project-task.adoc

```diff
@@ -2,42 +2,38 @@
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
-:page-aliases: apikit-4-update-wsdl-task.adoc
 
-This process walks you through the creation of a new Mule project based on a SOAP API specification.
+Create a new Mule project based on a SOAP API specification.
 
-Before proceeding, ensure that you meet the xref:apikit-4-soap-prerequisites-task.adoc[Prerequisites for Using APIkit for SOAP].
+Before proceeding, ensure that you meet the xref:apikit-4-soap-prerequisites-task.adoc[prerequisites for using APIkit for SOAP].
 
-. Under the File menu, select *New* > *Mule Project*.
+. In Anypoint Studio, from *File*, select *New* > *Mule Project*.
 +
-image::create-new-mule-project-soap.png["New Mule project window highlighting the project-name field, the runtime engine, the from design center or local files tab, and the expand icon for browsing files."]
+image::create-new-mule-project-soap.png["New Mule project window highlighting the project-name field, the runtime engine, the from design center or local files tab, and the expand icon for browsing files"]
 
 [calloutlist]
 . In *Project Name*, enter a name for the project. +
-For example, _soap-implementation_.
+For example, `soap-implementation`.
 . Select the Mule runtime engine version. +
 For example, `Mule Server 4.2.2 EE`.
-. Under *API Specification*, select the  *From Design Center or Local Files* tab.
-. Select the expand icon: image:expand-icon.png[30,35] and select *Browse files...*.
+. In *API Specification*, select *From Design Center or Local Files*.
+. Select *Browse files* from the *More actions* menu.
 
 [start=2]
-. Select a service and port from the drop-down menus or accept the defaults. For this example, accept the defaults: Select `TshirtService` and `TshirtServicePort`.
+. Select a service and port. For this example, accept the default values, `TshirtService` and `TshirtServicePort`.
 +
-image::create-apikit-soap-project.png["API specification window highlights the service and the port from the drop-down menus."]
+image::create-apikit-soap-project.png["API specification window highlighting the service and the port from the menu"]
```

Comment:
global - highlighting > that highlights
containing > that contains

Using gerunds in this way is not as clear. It can be ambiguous.

Reply:
OK! Thanks!